components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -33,11 +33,19 @@ const useStyles = makeStyles(() =>
         noCursor: {
             cursor: 'inherit',
         },
+        bottomSheet: {
+            width: '100%',
+            maxWidth: 600,
+            margin: 'auto',
+            userSelect: 'none',
+            cursor: 'pointer',
```

Comment:
I'm not sure you want to assume clickable (pointer) for the entire bottomsheet. Probably better to let the individual menu items determine that, right?